### PR TITLE
Header update

### PR DESCRIPTION
**Pull request by jlusds**
This PR ... 

## Changes
- Made a title change because it was confusing before

## Checklist
- [X] Tested locally?
- [X] Ran `quickTest all`? = no not needed because content
- [X] Ran `./prime test` against local Docker ReportStream container?
- [ ] Downloaded a file from http://localhost:7071/api/download
- [ ] Updated the release notes? 
- [ ] Added tests?
- [ ] Did you check for sensitive data, and remove any? 
- [ ] Does logging contain sensitive data?  
- [ ] Are additional approvals needed for this change?
- [ ] Are there potential vulnerabilities or licensing issues with any new dependencies introduced?

## Security
*List potential security threats and mitigations in the PR*

## Fixes
*List GitHub issues this PR fixes*
- #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

### CODE REVIEW

File: frontend/src/_data/content/index.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-title: ReportStream makes it really easier to share public health data
+title: ReportStream makes it really REALLY easier to share public health data
```
Comment (drew-usds) [inline frontend/src/_data/content/index.yaml:2]: Remove all caps

File: frontend/src/_data/content/index.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: ReportStream makes it really easier to share public health data
```
Comment (drew-usds) [inline frontend/src/_data/content/index.yaml:2]: Test this other thing

**Review by drew-usds — CHANGES_REQUESTED** [review]
Some content edits

File: frontend/src/_data/content/index.yaml (hunk 2 above)
Comment (jlusds) [reply frontend/src/_data/content/index.yaml:2]: I made the update you asked for

Comment (jlusds) [reply frontend/src/_data/content/index.yaml:2]: I'm still not sure if bold is better than all caps

Comment (jlusds) [reply frontend/src/_data/content/index.yaml:2]: verbally chatted on 3/23, drew is cool with bold

**Review by drew-usds — APPROVED** [review]
Looks good! 👍

**Pull request merged**